requirements.txt

```diff
@@ -0,0 +1 @@
+numpy~=1.19.5
```

Comment:
you should add here pandas
I haven't spoted that you used numpy

Reply:
good catch! :D